wordpress-mobile/WordPress-Android
# Blaze: Fixes Bulleted text and Image resource

Fixes #18049 

## Description 
This PR fixes the 
- Bulleted text style not being visible in the dark mode
- The incorrect image on the overlay 

| Light Mode | Dark mode  |
|---|---|
| ![screenshot_light_blaze-overlays_](https://user-images.githubusercontent.com/17463767/223115389-b2ff28c7-3c29-4cba-8ef8-d6895bd919f3.png)  |![screenshot_blaze_overlay_after_change_](https://user-images.githubusercontent.com/17463767/223115410-409e53bf-b1f5-4baf-8b26-2528540e3c13.png)| 

## To test:
### Pre-requisistes 
- Go to jetpack app 
- Login with a wp account with Administor privileges 
- Go to **app settings** → **Debug settings** 
- enable feature flag → `blaze`

### Test 
- Go to posts → Published Posts → Promote with blaze
- Verify that the overlay is shown 
- Verify that the Blaze icon on the top is correct 
- Verify that the Bulleted text is visible in Darkmode and light mode

## Regression Notes
1. Potential unintended areas of impact


2. What I did to test those areas of impact (or what existing automated tests I relied on)


3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
